Py_FS/wrapper/nature_inspired/ALO.py

```diff
@@ -0,0 +1,167 @@
+"""
+Programmer: Rishav Pramanik
+Date of Development: 20/08/2021
+Antlion Optimizer
+"""
+import numpy as np
+from Py_FS.wrapper.nature_inspired.algorithm import Algorithm
+from Py_FS.wrapper.nature_inspired._transfer_functions import get_trans_function
+from sklearn import datasets
+
+class ALO(Algorithm):
+    def __init__(self,
+                 num_agents,
+                 max_iter,
+                 train_data,
+                 train_label,
+                 test_data=None,
+                 test_label=None,
+                 val_size=20,
```

Comment:
What is the use of the val_size variable? Didn't find it anywhere else.

Reply:
Good point, removed!